apps/voting/test/voting.js

```diff
@@ -174,189 +139,234 @@ contract('Voting App', accounts => {
             assert.equal(await voting.minAcceptQuorumPct(), 1, 'should have changed acceptance quorum')
         })
 
-        it('fails changing minimum acceptance quorum to zero', async () => {
-            return assertRevert(async () => {
-                await voting.changeMinAcceptQuorumPct(0)
-            })
-        })
-
         it('fails changing minimum acceptance quorum to greater than min support', async () => {
             return assertRevert(async () => {
                 await voting.changeMinAcceptQuorumPct(neededSupport.plus(1))
             })
         })
 
-        context('creating vote', () => {
-            let script, voteId
+    })
+
+    for (const decimals of [0, 2, 10, 18, 36]) {
```

Comment:
Nice one. We probably don't need to test all these cases though, I would test a maximum of 3, otherwise it is going to take for ever to run voting tests